packages/maskbook/src/components/InjectedComponents/NFT/NFTAvatar.tsx

```diff
@@ -224,6 +224,27 @@ export interface AvatarMetaDB {
     symbol: string
 }
 
+interface NFT {
+    amount: string
+    symbol: string
+    name: string
+    image: string
+}
+
+const NFTAvatarCache = new Map<string, AvatarMetaDB>()
+const NFTCache = new Map<string, NFT>()
+
+function getNFTCache(userId: string) {
+    if (NFTCache.has(userId)) {
```

Comment:
```suggestion
    return NFTCache.get(userId)
```

Reply:
why not use `lodash.memoize`?

see https://lodash.com/docs/4.17.15#memoize